### DISCUSSION TITLE
Turkish language support

### DISCUSSION THREAD

**Discussion by seymasa** [explosion/spaCy#6554] [Language Support]
Why is the Turkish language support we expected since 2018 not coming out?
**I'm ready to do whatever it takes to support this!**

**Comment by svlandeg** [comment]
There was some discussion in [this thread](https://github.com/explosion/spaCy/discussions/2617), perhaps you can find some collaborators there to get this going?

**Comment by svlandeg** [comment]
Update: https://github.com/explosion/spaCy/discussions/12390 introduces Turkish models trained by a community member!
